api/envoy/config/listener/v3/listener.proto

```diff
@@ -94,6 +95,11 @@ message Listener {
 
       // If specified, the listener will use the exact connection balancer.
       ExactBalance exact_balance = 1;
```

Comment:
Not sure if you guys already discussed this, whether we should deprecate this field, then move the exact_balance as an extension also.

Reply:
That would be optimal but IMO it's not worth the churn.